devops/helm/crunchy-postgres/values-0c27fb-dev.yaml

```diff
@@ -15,11 +15,15 @@ standby:
   # If you want to recover from PVC, use repo1. If you want to recover from S3, use repo2
   repoName: repo1
 
+# enable this to go back to a specific timestamp in history in the current cluster.
+# follow https://access.crunchydata.com/documentation/postgres-operator/5.2.0/tutorial/disaster-recovery/
+# Perform an In-Place Point-in-time-Recovery (PITR)
+# this configuration for restore is a dummy configuration. Actual values used for the restore are provided to make through the github actions workflow
 restore:
-  enabled: "${RESTORE_ENABLED}"
-  repoName: repo1
+  enabled: false
```

Comment:
The entire restore settings can be removed from the values-`env`.yaml and kept only in the values.yaml. Does it make sense?